modules/nicknames/renameMember.js

```diff
@@ -0,0 +1,63 @@
+const {truncate} = require('../../src/functions/helpers');
+const {localize} = require('../../src/functions/localize');
+
+renameMember = async function (client, guildMember) {
+    const roles = client.configurations['nicknames']['config'];
+    const moduleModel = client.models['nicknames']['User'];
+
+    if (guildMember.guild.ownerId === guildMember.id) {
+        client.logger.error('[nicknames] ' + localize('nicknames', 'owner-cannot-be-renamed', {u: guildMember.user.username}))
+        return;
+    }
+
```

Comment:
So this will always display the message, could we move this when we have confirmed that the nickname does not meet the expectations?


```suggestion
```